sklearn/gaussian_process/_gpc.py

```diff
@@ -606,6 +606,12 @@ def optimizer(obj_func, initial_theta, bounds):
 
         .. versionadded:: 0.24
 
+    See Also
+    --------
+    GaussianProcessRegressor : Regressor module for Gaussian Proccess.
+    kernels.Exponentiation : Exponentiation kernel for Gaussian Proccess.
```

Comment:
I would only put the regressor and do not mention the kernels

Reply:
Of course, done